colabs/Wczytanie_danych.ipynb

```diff
@@ -448,15 +435,401 @@
+        "  - knownForTitles (tytuły produkcji, z których znany jets aktor - może posłużyć do porównania aktorów na podstawie tych samych lub podobnych produkcji, z których są znani)"
+      ],
+      "metadata": {
+        "id": "MGd5Hi-VBF9h"
+      }
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "df_title_basic_selected = df_title_basic.select([\"tconst\", \"titleType\", \"originalTitle\", \"isAdult\", \"genres\"])\n",
+        "df_title_principals_selected = df_title_principals.select([\"tconst\", \"nconst\", \"category\", \"characters\"])\n",
+        "df_name_basics_selected = df_name_basics.select([\"nconst\", \"primaryName\", \"birthYear\", \"deathYear\", \"knownForTitles\"])"
+      ],
+      "metadata": {
+        "id": "OKWe9JSKBC5q"
+      },
+      "execution_count": 22,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "# Złączmy tabele df_name_basics, df_title_principals i df_title_basic"
+      ],
+      "metadata": {
+        "id": "eMIHo59PBNtx"
+      }
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "print(\"df_name_basics_selected dataframe size: \", (df_name_basics_selected.count(), len(df_name_basics_selected.columns)))\n",
+        "print(\"df_title_principals_selected dataframe size: \", (df_title_principals_selected.count(), len(df_title_principals_selected.columns)))\n",
+        "print(\"df_title_basic_selected dataframe size: \", (df_title_basic_selected.count(), len(df_title_basic_selected.columns)))\n",
+        "data = df_name_basics_selected.join(df_title_principals_selected, \"nconst\", \"left\")\n",
+        "print(\"joined dataframe size: \", (data.count(), len(data.columns)))\n",
+        "data = data.join(df_title_basic_selected, \"tconst\", \"left\")\n",
+        "print(\"joined dataframe size: \", (data.count(), len(data.columns)))"
+      ],
+      "metadata": {
+        "id": "zMqqAJ7zBJWL",
+        "outputId": "4d7dec89-79e4-4d61-98b2-8295666b7df4",
+        "colab": {
+          "base_uri": "https://localhost:8080/"
+        }
+      },
+      "execution_count": 25,
+      "outputs": [
+        {
+          "output_type": "stream",
+          "name": "stdout",
+          "text": [
+            "df_name_basics_selected dataframe size:  (12060121, 5)\n",
+            "df_title_principals_selected dataframe size:  (52924405, 4)\n",
+            "df_title_basic_selected dataframe size:  (9354673, 5)\n",
+            "joined dataframe size:  (60141511, 8)\n",
+            "joined dataframe size:  (60141511, 12)\n"
+          ]
+        }
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "## Wybierzmy z tabeli tylko aktorów i aktorki"
+      ],
+      "metadata": {
+        "id": "Mhd-FvVZBQz7"
+      }
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "data = data.filter((data.category == \"actor\") | (data.category == \"actress\"))"
```

Comment:
chyba lepiej jest to filtrowanie zrobić przed joinem

Reply:
zmieniłam kolejność, ale wiązała się z tym zmiana rodzaju łączenia dwóch pierwszych tabel na "inner".